src/lambda_alerting.py

```diff
@@ -60,6 +60,7 @@ def lambda_handler(event, context):
     event_result = mapper.get_event_result()
     resource_name = mapper.get_resource_name()
     event_status = mapper.get_resource_state()
+    execution_info_url = mapper.get_execution_info_url(resource_type, resource_name)
```

Comment:
maybe we can store resource_type in mapper's __init__ and not pass it here

Reply:
yes, sure